src/vehicle.cpp

```diff
@@ -701,7 +701,7 @@ void vehicle::smash( float hp_percent_loss_min, float hp_percent_loss_max,
 int vehicle::lift_strength() const
```

Comment:
The return type of this function is still `int`, which the `int64` gets converted to, which can't be intended.

Reply:
This being int is fine, it is related to the strength of a character, not to a mass (it is calculated from a mass).